install/kubernetes/Makefile

```diff
@@ -26,6 +26,7 @@ DEV_VERSION_REGEX := '[0-9]\+\.[0-9]\+-dev'
 CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_PULLPOLICY_REGEX := '\([pP]ullPolicy:\) .*'
 
+SED := sed
```

Comment:
nit: can you move this to Makefile.defs ? so that later we can easily swap sed command in main Makefile as well.

Reply:
and should it be `SED ?= sed` ?

Reply:
Updated.